docs/synapse/devopsguide.rst

```diff
@@ -981,6 +981,124 @@ this to add a user named ``foouser`` with the uid 1234::
     Successfully built 21a12f395462
     Successfully tagged vertexproject/synapse-aha:v2.113.0-foouser
 
+Configure Custom CA Certificates for HTTP Requests
+--------------------------------------------------
+
+The Cortex can be configured to use additional CA certificates when making HTTP
+requests via Storm. To do this, you need to provide the certificates (in DER or
```

Comment:
Have you tested with DER format? I've only tested with PEM format and the sslcontext docs note it is PEM encoded: https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_verify_locations

Reply:
Good call. DER does not work. Updated text.